src/arduino_lmic_hal_boards.h

```diff
@@ -3,13 +3,13 @@
 Module:  arduino_lmic_hal_boards.h
 
 Function:
-        Arduino-LMIC C++ HAL pinmaps for various boards
```

Comment:
Did you intend to change whitespace here? Please don't change unless you convert the entire file, and it really should be a separate pull request.

Reply:
Sorry, this whitespace was changed by mistake and not by intend. I will revert this change.